Zend/zend_objects_API.h

```diff
@@ -45,6 +45,7 @@ typedef struct _zend_objects_store {
 	uint32_t top;
 	uint32_t size;
 	int free_list_head;
+	char no_reuse; /* to be set to true when shutting down, to avoid missing dtor call on objects spawned by another dtor */
```

Comment:
This is an ABI break on 32-bit platforms, so this change could only go into master as-is.